pkg/appregistry/package.go

```diff
@@ -0,0 +1,28 @@
+package appregistry
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Package struct {
+	Name    string
```

Comment:
readability nit: we have established in registry / olm that "package" is specifically _not_ referring to a particular release, so I would rename this (and the other references to it) to indicate that it represents a particular release and not a package. 

Elsewhere in registry we refer to these as "ChannelEntries", but that may not fit this case since channel info is not used in appregistry.

Reply:
Sure! This PR isn't a huge priority for our team so I will try and get around to it soon!